browser/renderer_context_menu/test/BUILD.gn

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
AFAIK, we want to put test file in the same directory of its target file instead of adding separated `test` dir.
cc @goodov